src/main/java/org/openstreetmap/atlas/geography/atlas/complete/EmptyAtlas.java

```diff
@@ -213,6 +212,18 @@ public AtlasEntity entity(final long identifier, final ItemType type)
         }
     }
 
+    @Override
+    public Iterable<AtlasEntity> getGeoJsonObjects()
+    {
+        throw new UnsupportedOperationException();
```

Comment:
Please consider returning an empty list here.

Reply:
I am open to it, but I am following the pattern of the other methods in the EmptyAtlas.java and am not sure about breaking from that convention.